libdnf/rpm/solv/id_queue.hpp

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright (C) 2017-2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_RPM_SOLV_ID_QUEUE_HPP
+#define LIBDNF_RPM_SOLV_ID_QUEUE_HPP
+
+#include <utility>
+
+extern "C" {
+#include <solv/queue.h>
+}
+
+namespace libdnf::rpm::solv {
+
+struct IdQueue {
+public:
+    IdQueue();
+    IdQueue(const IdQueue & src);
+    IdQueue(IdQueue && src);
+    explicit IdQueue(const Queue & src);
+    ~IdQueue();
+
+    bool operator==(const IdQueue & other) const;
+    bool operator!=(const IdQueue & other) const;
+    IdQueue & operator=(const IdQueue & src);
+    IdQueue & operator=(IdQueue && src) noexcept;
+    Id operator[](int index) const;
+
+    void push_back(Id id);
+    void push_back(Id id1, Id id2);
```

Comment:
`push_back(Id id)` is obvious, but why there's the 2nd version with 2 Ids? Could you document what it does?

Reply:
It is used for a libsolv job. I will add a note.